contrib/test.sh

```diff
@@ -66,3 +66,20 @@ if [ "$DO_BENCH" = true ]
 then
     cargo bench --features="unstable compiler"
 fi
+
+# Run Integration tests if told so
+if [ -n "$BITCOINVERSION" ]; then
+    set -e
+    cd integration_test
+    rm -rf bitcoin-$BITCOINVERSION-x86_64-linux-gnu.tar.gz  
+    wget https://bitcoincore.org/bin/bitcoin-core-$BITCOINVERSION/bitcoin-$BITCOINVERSION-x86_64-linux-gnu.tar.gz
```

Comment:
with curl you can pipe directly to avoid saving an external file (and removing it), something like:
https://bitcoincore.org/bin/bitcoin-core-$BITCOINVERSION/bitcoin-$BITCOINVERSION-x86_64-linux-gnu.tar.gz | tar -xvz 

optionally to speed up a bit you can also extract only the bitcoind executable, but in that case you have to change the shasum verification part so maybe not worthy

Reply:
Thanks, updated the PR with both suggestions.

Reply:
This could also be done with `wget -O - ... | tar ...`